meta/store.go

```diff
@@ -394,6 +397,30 @@ func (s *Store) enableRemoteRaft() error {
 	return s.changeState(rr)
 }
 
+func (s *Store) checkRaftState() error {
```

Comment:
This is for when the server is already running, and then is promoted to a raft peer.  The only way to determine that state is if you are in the peers and have a remoteRaft.  If you are in the peers and have a localRaft, there is nothing to do.